addons/mail/static/tests/chatter/web/follow_button.test.js

```diff
@@ -23,7 +23,9 @@ test("base rendering follow and unfollow button", async () => {
     await contains(".o-mail-Followers-counter", { text: "1" });
     await contains("[title='Show Followers'] .fa-user");
     await click("[title='Show Followers']");
-    await contains("[title='Edit subscription']");
+    await click("[title='Edit subscription']");
+    await contains(".o-mail-Followers-dropdown", { count: 0 });
+    await click("[title='Show Followers']");
```

Comment:
Don't use a selector in a negative assertion without a matching positive assertion with same selector.

This is important because maintainability of negative assertion is very poor. This is improved drastically by having matching positive assertion, so that test fails when the selector needs to be updated, and since we have to change this test the likelihood to adapt negative assertions is high.

When the positive assertion is in another test, the likelihood is very low, because we only adapt tests that fail, not test that pass but normally shouldn't.